documentation/modules/ROOT/examples/abstract-property-sample.ttl

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Robert Bosch Manufacturing Solutions GmbH
+
+# See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
+
+# This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+# If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+# SPDX-License-Identifier: MPL-2.0
+
+@prefix : <urn:samm:com.mycompany.myapplication:1.0.0#> .
```

Comment:
But this will only be adapted in the other applications (SDK, ...) from phase 2?

Reply:
Yes, supporting this will need to be implemented downstream. This is not something that we can address here.